### ISSUE TITLE
Gflow: Shuffled Pairs List

### ISSUE THREAD

**Issue by Pbleonard** [Pbleonard/GFlow#3]
Include snippet that allows user to create all pairs .txt from which a random selection can then be made. Not sure about status? Already included?

**Comment by eduffy** [comment]
This feature is now builtin.  The patch 3593cd6 adds a new flag `-shuffle_node_list` to shuffle the node list.  It does not, however, truncate the list to a predefined number of pairs (like you can do with `shuf`).

**Comment by Pbleonard** [comment]
@eduffy: It looks like the flat is actually `-shuffle_node_pairs`

**Comment by eduffy** [comment]
Flat=fault?
What makes you say that? This is the MHI test?

sent from my mobile.

**Comment by Pbleonard** [comment]
@eduffy  I isolated the problem on Palmetto I was telling you about with new flags. Will not accept new flag `shuffle_node_pairs`. I went back to this location to make sure that was the correct syntax in commit. It appears to be. However in the comment above you list `shuffle_node_list`. Regardless, even with the correct syntax the flag is creating some problem on Palmetto. Works fine locally though so did not re-open.
